src/sentry/utils/eventuser.py

```diff
@@ -72,7 +73,10 @@ def from_event(event: Event) -> EventUser:
             project_id=event.project_id if event else None,
             email=event.data.get("user", {}).get("email") if event else None,
             username=event.data.get("user", {}).get("username") if event else None,
-            name=event.data.get("user", {}).get("name") if event else None,
+            name=event.data.get("user", {}).get("name")
+            or event.data.get("user", {}).get("username")
```

Comment:
Hmm, I'm not totally sure we should put username here, should be ok to leave it blank if not provided (unless there's an example elsewhere where we substitute in username)